package.json

```diff
@@ -6,23 +6,28 @@
   "module": "./src/fake-xml-http-request.js",
   "repository": "https://github.com/trek/FakeXMLHttpRequest.git",
   "scripts": {
-    "test": "esperanto -i src/fake-xml-http-request.js -t umd -b -n FakeXMLHttpRequest -o fake_xml_http_request.js && ./node_modules/karma/bin/karma start --single-run --browsers PhantomJS"
+    "start": "karma start",
+    "build": "rollup src/fake-xml-http-request.js --file fake_xml_http_request.js --format umd --name 'FakeXMLHttpRequest' --sourcemap true",
+    "test": "npm run build && karma start --single-run --browsers PhantomJS"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=14"
```

Comment:
We should remove engines.node if not used in node at all, otherwise yarn/npm would complain in dependents.

Reply:
Removed.